### PR TITLE
thuang-math-file-size

### PR DESCRIPTION
**Pull request by tihuan**
#### Reviewers
**Functional:** 
@seve 

**Readability:** 
@Bento007 

---

## Changes
- modify
Change the file size calculation to use the correct formula
E.g., `Math.round(402210784 / 2 ** 20)` outputs `384MB`

PTAL thank you!

### CODE REVIEW

File: frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/Details/index.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface Props {
   isLoading: boolean;
 }
 
-const MEGA_BYTES = 1000 * 1000;
+const MEGA_BYTES = 2 * 20;
```
Comment (seve) [inline frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/Details/index.tsx:14]: Shouldn't this be:
```suggestion
const MEGA_BYTES = 2 ** 20;
```

**Review by seve — DISMISSED** [review]

**Review by Bento007 — COMMENTED** [review]
With Seve's change, I approve

**Comment by tihuan** [comment]
Oh yes 😆 Thank you so much for catching that, @seve ! The one time I skipped testing the UI due to handling prod deployment at the same time lol Never again 🤣 

Thanks both!

**Comment by tihuan** [comment]
Ooh I see, so we're just missing an LGTM 😆 I'll force merge this one. Thanks all!

**Pull request merged**